_posts/2021-05-12-244.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 244
+author: kkoukiou
+date: 2021-05-12
+tags: cockpit linux
+slug: cockpit-244
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 244 and Cockpit-Machines 244.
+
+## Shell: No opportunistic sudo invocation
```

Comment:
I feel this is a strange headline:

- It's weird to have this be a negative with "no"
- "opportunistic" is odd
- "invocation" is also odd (unless you're in a church)

Reply:
Ok this comment is super funny, but does not help me much.

When you 'call' sudo, or any command it's invocation. It's a common word to use I feel.

Slightly knowing what that change is about, I think this is actually a short but descriptive header.

Reply:
https://en.wikipedia.org/wiki/Invocation:

> An invocation (from the Latin verb invocare "to call on, invoke, to give") may take the form of:
> - Supplication, prayer or spell.
> - A form of possession.
> - Command or conjuration.
> - Self-identification with certain spirits.

Reply:
Hmm let me quickly edit wikipedia to add how else it can be used :)

https://linux.die.net/man/8/sudo if you search for `invok` there are 11 results on the sudo man page.

Reply:
If you want to use a form of the word that isn't so supernatural:

```suggestion
## Shell: Sudo is no longer invoked unless needed
```

But that's still a bit problematic.

```suggestion
## Shell: Sudo is used only when necessary
```

I don't know. What do you think?

Reply:
But that last one makes people think "was sudo used willy-nilly?" (that is for no good reason whatsoever, haphazardly, in a sloppy way, etc.)

Reply:
'Unless needed' or 'when necessary' implies that it will be always invoked automatically when needed, but it's not the case. The user needs to turn on the 'Admin access'

Reply:
What about "Shell: sudo will only be invoked when explicitly requested"

Reply:
ok!

Reply:
Sure, that works for me.

Reply:
Oh wait. Minor revision:

Shell: sudo is invoked only when explicitly requested

Reply:
Fixed.